packages/graphile-build-pg/src/plugins/PgTypesPlugin.js

```diff
@@ -26,44 +25,6 @@ function parseInterval(str) {
   return result;
 }
 
-const pgRangeParser = {
-  parse(str) {
-    const parts = str.split(",");
-    if (parts.length !== 2) {
-      throw new Error("Invalid daterange");
-    }
-
-    return {
-      start:
-        parts[0].length > 1
-          ? {
-              inclusive: parts[0][0] === "[",
-              value: parts[0].slice(1),
-            }
-          : null,
-      end:
-        parts[1].length > 1
-          ? {
-              inclusive: parts[1][parts[1].length - 1] === "]",
-              value: parts[1].slice(0, -1),
-            }
-          : null,
-    };
-  },
-
-  serialize({ start, end }) {
-    const inclusivity = {
-      true: "[]",
-      false: "()",
-    };
-
-    return [
-      start ? inclusivity[start.inclusive][0] + start.value : "[",
-      end ? end.value + inclusivity[end.inclusive][1] : "]",
-    ].join(",");
-  },
```

Comment:
We seem to have lost the serialize capability for ranges. Do we not have any tests for this?!

Reply:
Oh, whoops, I replaced this in the mapper but never deleted the dead code. Fine 👍